crates/ere-pico/src/client.rs

```diff
@@ -0,0 +1,99 @@
+// Copied and modified from https://github.com/brevis-network/pico/blob/v1.1.7/sdk/sdk/src/client.rs.
+// The `EmbedProver` is removed because we don't need the proof to be verified
+// on chain.
```

Comment:
Worth creating an issue to remember trying to avoid/remove this eventually?

I think might be useful to ping Brevis to provide proper functionality via the SDK.
(I added you to a TG group `Brevis <> EF` which can be a good medium to ask this or anything, if you want)

Reply:
Make sense, created an issue in https://github.com/eth-act/ere/issues/140, and will ask if the Brevis team can help us on that